src/algorithm/contact-dynamics.hpp

```diff
@@ -144,23 +144,46 @@ namespace pinocchio
       return forwardDynamics(model,data,tau,J,gamma,inv_damping);
   }
 
-  
   ///
   /// \brief Computes the inverse of the KKT matrix for dynamics with contact constraints, [[M JT], [J 0]].
```

Comment:
Is this description alright? As far I understood, we should need to run forward or impulse dynamics, right?

Reply:
Yes, stupid copy/paste from my side.